sdk/cosmos/ci.spark.yml

```diff
@@ -0,0 +1,16 @@
+trigger: none
+
+variables:
+  - template: ../../eng/pipelines/templates/variables/globals.yml
+  - name: AdditionalArgs
+    value: ''
+
+stages:
+  - ${{ if or(eq(variables['Build.Reason'], 'Manual'), eq(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
Is this check even necessary? You are basically saying "don't run this as a scheduled build" but since it's its own pipeline now, we can just remove the schedule entirely.